test_api_performance.py

```
@@ -0,0 +1,45 @@
+                first_project = data['data'][0]
+                print(f"\nFirst project: {first_project['name']}")
+                print(f"Versions count: {len(first_project['versions'])}")
+                if first_project['versions']:
+                    print("WARNING: Version data is still being loaded!")
+                    print(f"First version: {first_project['versions'][0]['version']}")
+                else:
+                    print("✓ No version data loaded (good for performance)")
+        else:
+            print(f"Error: {response.text}")
+
+    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)